### DISCUSSION TITLE
xmake install and pkg-config

### DISCUSSION THREAD

**Discussion by maluoi** [xmake-io/xmake#1301] [Q&A] [answered]
First of all, thanks so much for creating xmake, it's easily the most enjoyable build tool I've worked with yet!

I'm building a cross-platform library, StereoKit, and I have some community members that are trying to add it as a dependency in their own projects on Linux. They're using pkg-config to locate and reference their dependencies, but `xmake install` for StereoKit doesn't currently output any pkg-config information.

I've looked around in docs and samples for how to output the pkg-config information, and I haven't quite managed to pin it down. Do you have any tips or pointers here for how to go about doing this? My build file is [here](https://github.com/maluoi/StereoKit/blob/1fdf39dd900a837b4df61dd10fa69313c1e4d424/xmake.lua) if that's helpful.

**Comment by waruqi** [comment, marked as answer]
You can write a xxx.pc file in `after_install()`, like this https://github.com/xmake-io/xmake/blob/a1ebb6a1b37dbcca45da44cb1b9a4f0341a2684a/xmake/modules/private/action/require/impl/actions/install.lua#L77-L91

or wait this feature https://github.com/xmake-io/xmake/issues/1268

**Reply by maluoi** [reply]
Oh wow, that'll totally work! Thanks a bunch :)
